frontstage/common/authorisation.py

```diff
@@ -58,12 +58,11 @@ def extract_session_wrapper(*args, **kwargs):
                 raise JWTValidationError
 
             if app.config['VALIDATE_JWT']:
-                valid = validate(jwt)
-            if valid:
-                session_handler.update_session()
-                return original_function(jwt, *args, **kwargs)
-            else:
-                logger.warning('Token is not valid for this request')
-                raise JWTValidationError
+                if validate(jwt):
+                    session_handler.update_session()
+                    return original_function(jwt, *args, **kwargs)
+                else:
```

Comment:
We don't need this else as there's a return in the if statement

Reply:
What about if validate(jwt) is False?